assets/package.json

```diff
@@ -4,19 +4,19 @@
   "private": true,
   "engine-strict": true,
   "engines": {
-    "node": "^14.18",
-    "npm": "^6.14"
+    "node": "^16.13",
+    "npm": "^8.1"
```

Comment:
I believe we can skip the npm information here now as it is bundled with Node, if I am not mistaken

Reply:
We kind of need it for the CI process to run on Github Actions…